src/recoilState/appState.ts

```diff
@@ -136,15 +136,24 @@ export const rMyCircles = selector<ICircle[]>({
   key: 'rMyCircles',
   get: ({ get }: IRecoilGetParams) => {
     const myProfile = get(rMyProfile);
-    return (
-      myProfile?.users
-        ?.map((u) => createCircleWithDefaults(u.circle))
-        .sort((a, b) => a.protocol_id - b.protocol_id) ?? []
-    );
+    return (myProfile?.users ?? [])
```

Comment:
style suggestion: `if (!myProfile?.users) return [];` and then `return myProfile?.users.map...`

Reply:
thats more lines